docs/controlling-fixtures/using-the-select-buttons-and-wheels.md

```diff
@@ -139,15 +155,18 @@ you hold down the \<Clear\> button, then more options are available.
     fixtures will be cleared. This is useful if you want to clear
     specific fixtures.
 
--   \[Individual Attributes\] allows you to clear individual attributes
-    from the Programmer. When you press the softkey, you are given a
-    list of attributes in the Programmer -- press the appropriate
-    softkey to clear that attribute.
+-   \[Individual Attributes\] gives you a list of attributes currently 
+	in the Programmer - press the appropriate softkey to clear that attribute. This
+	lets you clear individual attributes rather than the whole group, for example
+	you could clear Pan but leave Tilt. You can press the attribute buttons to filter
+	what's shown in the list.
 
 -   \[Clear All Programmers\] will clear all programmers that are
     currently active on the console, including Multi-user programmers
     from other users, and the Titan Remote programmer.
 
+### Clear options menu
+
 The options in the "Clear Options" submenu are:
```

Comment:
It might also be worth pointing out that a number of these (including Action Precedence) are actually User Settings and as such show up in the user settings window / menu and save/load with your user, and are made available here for convenience